decentraland/unity-renderer
# feat: lambas sentry transactions

## What does this PR change?

Closes #3979 

- Added a generic way of tracking `UnityWebRequest` and reporting its result to Sentry
- Applied `WebRequestMonitor` to `LambdasService`, in the future it can be further expanded to any `WebRequest` if have proven useful

...

## How to test the changes?
1. Make Extended Passport Requests (Open Other Users' Profiles)
2. Go to https://sentry.io/organizations/decentraland/performance/
3. You should see reported Requests (keep in mind that the current rate is 5% so only 5% of requests will appear there)
4. Observe "GET nfts/names" and "GET nfts/lands" are reported like this:
<img width="530" alt="image" src="https://user-images.githubusercontent.com/118179774/213990210-cab49903-cf6d-41d0-bab2-0cca51da94ee.png">


1. Go to: https://play.decentraland.zone/?renderer-branch={branch_name}&{desired_url_params}
6. ...

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
